### PR TITLE
migrated the config attributes spec to batch

### PR DESCRIPTION
**Pull request by jarroyo28**
Migrated the ConfigurationAttributesSpec to batch.
There are TODOs in the code to determine whether some methods are required or not.

### CODE REVIEW

File: supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala [hunk 1]
```diff
@@ -0,0 +1,468 @@
+package cromwell.backend.google.pipelines.batch
+
+import cats.data.Validated.{Invalid, Valid}
+import cats.syntax.validated._
+import com.typesafe.config.{Config, ConfigFactory}
+import common.assertion.CromwellTimeoutSpec
+import common.exception.MessageAggregation
+import cromwell.backend.google.pipelines.batch.GcpBatchTestConfig.BatchGlobalConfig
+import cromwell.backend.google.pipelines.common.PipelinesApiConfigurationAttributes._
```
Comment (dspeck1) [inline supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala]: Please remove reference to common. Replace with batch configuration attributes.

**Review by dspeck1 — CHANGES_REQUESTED** [review]
Overall looks good please remove import from common

File: supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala (hunk 1 above)
Comment (jarroyo28) [reply supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala]: I removed references to pipelines.common in new commit

File: supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+//import java.net.URL
+import scala.concurrent.duration._
+
+// TODO: Need to replace CromwellTimeoutSpec with appropriate class from batch
+class GcpBatchConfigurationAttributesSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers
+  with TableDrivenPropertyChecks {
+
+  behavior of "PipelinesApiAttributes"
```
Comment (dspeck1) [inline supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala]: Change wording to Batch

File: supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+
+  val googleConfig: GoogleConfiguration = GoogleConfiguration(BatchGlobalConfig)
+  val runtimeConfig: Config = ConfigFactory.load()
+
+  it should "parse correct PAPI config" in {
```
Comment (dspeck1) [inline supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala]: Change wording from PAPI to Batch

File: supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala (hunk 2 above)
Comment (jarroyo28) [reply supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala]: Wording has been changed to 'GcpBatchAttributes'

File: supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala (hunk 3 above)
Comment (jarroyo28) [reply supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala]: wording has been changed to Batch

**Review by dspeck1 — APPROVED** [review]
looks good.

**Pull request merged**